[2025-09-04T17:13:01.256+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-04T17:13:01.284+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: foreign_1_live.save_foreign_1 scheduled__2025-09-04T17:12:00+00:00 [queued]>
[2025-09-04T17:13:01.295+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: foreign_1_live.save_foreign_1 scheduled__2025-09-04T17:12:00+00:00 [queued]>
[2025-09-04T17:13:01.295+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 11
[2025-09-04T17:13:01.310+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): save_foreign_1> on 2025-09-04 17:12:00+00:00
[2025-09-04T17:13:01.318+0000] {standard_task_runner.py:63} INFO - Started process 3017349 to run task
[2025-09-04T17:13:01.322+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'foreign_1_live', 'save_foreign_1', 'scheduled__2025-09-04T17:12:00+00:00', '--job-id', '101455', '--raw', '--subdir', 'DAGS_FOLDER/dag_foreign_1_live.py', '--cfg-path', '/tmp/tmp8lk5_wjt']
[2025-09-04T17:13:01.326+0000] {standard_task_runner.py:91} INFO - Job 101455: Subtask save_foreign_1
[2025-09-04T17:13:01.385+0000] {task_command.py:426} INFO - Running <TaskInstance: foreign_1_live.save_foreign_1 scheduled__2025-09-04T17:12:00+00:00 [running]> on host 1877907bc043
[2025-09-04T17:13:01.501+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='foreign_1_live' AIRFLOW_CTX_TASK_ID='save_foreign_1' AIRFLOW_CTX_EXECUTION_DATE='2025-09-04T17:12:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-04T17:12:00+00:00'
[2025-09-04T17:13:01.502+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-04T17:13:01.516+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-04T17:13:01.895+0000] {foreign_1.py:28} INFO - Đã lưu foreign_ALL
[2025-09-04T17:13:01.896+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-04T17:13:02.238+0000] {foreign_1.py:28} INFO - Đã lưu foreign_HOSE
[2025-09-04T17:13:02.239+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-04T17:13:02.629+0000] {foreign_1.py:28} INFO - Đã lưu foreign_HNX
[2025-09-04T17:13:02.631+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-04T17:13:03.694+0000] {foreign_1.py:28} INFO - Đã lưu foreign_UPCOM
[2025-09-04T17:13:03.694+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-09-04T17:13:03.695+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-04T17:13:03.712+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=foreign_1_live, task_id=save_foreign_1, run_id=scheduled__2025-09-04T17:12:00+00:00, execution_date=20250904T171200, start_date=20250904T171301, end_date=20250904T171303
[2025-09-04T17:13:03.780+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-04T17:13:03.820+0000] {taskinstance.py:3498} INFO - 4 downstream tasks scheduled from follow-on schedule check
[2025-09-04T17:13:03.822+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
